builder/azure/arm/artifact.go

```diff
@@ -249,13 +250,45 @@ func (*Artifact) Destroy() error {
 	return nil
 }
 
-func (a *Artifact) stateAtlasMetadata() interface{} {
-	metadata := make(map[string]string)
-	metadata["StorageAccountLocation"] = a.StorageAccountLocation
-	metadata["OSDiskUri"] = a.OSDiskUri
-	metadata["OSDiskUriReadOnlySas"] = a.OSDiskUriReadOnlySas
-	metadata["TemplateUri"] = a.TemplateUri
-	metadata["TemplateUriReadOnlySas"] = a.TemplateUriReadOnlySas
+func (a *Artifact) hcpPackerRegistryMetadata() *registryimage.Image {
+	var id, location string
+
+	if a.isManagedImage() {
+		id = a.ManagedImageId
+		location = a.ManagedImageLocation
+
+		labels := make(map[string]interface{})
+		labels["os_type"] = a.OSType
+		labels["managed_image_resourcegroup_name"] = a.ManagedImageResourceGroupName
+		labels["managed_image_name"] = a.ManagedImageName
+		if a.ManagedImageSharedImageGalleryId != "" {
+			labels["managed_image_sharedimagegallery_id"] = a.ManagedImageSharedImageGalleryId
```

Comment:
We probably also need the SIG name, as well as the SIG Image Name, and the SIG version.

Reply:
See PR #150